### ISSUE TITLE
About Evaluation 

### ISSUE THREAD

**Issue by big-chan** [UkcheolShin/ThermalMonoDepth#4]
Hello

I've seen this interesting research very well.

I'm running the evaluation code and checking the performance, but I don't know what kind of performance is correct to compare with the reported performance.

It is reported that the performance of the Indoor test set (Well-lit) is as follows.

```
Ours | 0.152 | 0.121 | 0.538 | 0.196 | 0.814 | 0.965 | 0.992
```
However, when the performance is measured using the uploaded pre-train model, the following performance is shown for the 
"**indoor robust_varing_well_it**" folder.
```
==> Evaluating depth result...
 Scaling ratios | med: 4.776 | std: 0.056
 Scaling ratios | mean: 4.764 +- std: 0.268

   abs_rel |   sq_rel |     rmse | rmse_log |       a1 |       a2 |       a3 | 
&   0.129  &   0.090  &   0.445  &   0.171  &   0.879  &   0.981  &   0.996  \\
```
Is it correct to look at the performance of the folder above to see the correct performance?
And in which folder should the "**Indoor test set (Low-/Zero-light)**" see the performance?

**Comment by UkcheolShin** [comment]
We utilize two sequences ("indoor_aggresive_local" and "indoor_robust_varying_well_lit") for indoor well-lit evaluation.
For the low-light evaluation, we use "indoor_robust_dark", "indoor_aggresive_dark", and "indoor_unstable_dark".
The final performance of each condition (i.e., well-lit and low-light) is averaged over the sequences.

For example, if the sequence "indoor_robust_varying_well_lit" has 100 samples and gets 0.129 in abs_rel error metric and "indoor_aggresive_local" has 50 samples and gets 0.120, then the final score is calculated to get averaged result as like (100*0.129+50*0.120)/150 =  0.126

**Comment by big-chan** [comment]
Thank you for reply :)
